src/main/java/uk/gov/hmcts/reform/docassembly/domain/DocAssemblyRequest.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.docassembly.domain;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+import java.util.Base64;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Builder
+@Getter
+@EqualsAndHashCode
```

Comment:
You may consider `@Value` instead of above

Reply:
True 👍 will fix